indexing-service/src/main/java/org/apache/druid/indexing/overlord/TaskQueue.java

```diff
@@ -175,6 +182,11 @@ public TaskQueue(
         config.getTaskCompleteHandlerNumThreads(),
         "TaskQueue-OnComplete-%d"
     );
+    // Add mixin to redact passwords.
```

Comment:
We should add a test case for this so that we donot miss this in a future code change.

Reply:
Added